<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerClass</class>
 <widget class="QMainWindow" name="PlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>352</width>
      <height>288</height>
     </rect>
    </property>
    <property name="text">
     <string>videoShow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardBn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>forward</string>
    </property>
   </widget>
   <widget class="QSlider" name="videoSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>361</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="backBn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>loadVideo</string>
    </property>
   </widget>
   <widget class="Phonon::SeekSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>361</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="player.qrc"/>
 </resources>
 <connections/>
</ui>
